.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: node_js
+before_install:
+- npm install -g npm@latest
```

Comment:
this might conflict with the iojs target, since iojs needs a special npm, right?